pkg/docker/containers-view.jsx

```diff
@@ -607,7 +610,7 @@ var ImageList = React.createClass({
         var columns = [
             { name: image.RepoTags[0], header: true },
             vulnerabilityColumn,
-            moment.unix(image.Created).fromNow(),
+            moment.unix(image.Created).calendar(),
```

Comment:
What does this change do?

Reply:
So `fromNow()` gives a relative timestamp (for instance "6 months ago"). `calendar()` gives preciser dates; if it's recent it will include hour/minutes, and just day/month/year if it's longer than a week ago.

Maybe we should get some more input from a designer on this choice.

Reply:
This makes sense, so let's use `calendar()`. Thanks!